src/core/CMakeLists.txt

```diff
@@ -67,8 +66,7 @@ add_library(
     include/mp-units/system_reference.h
     include/mp-units/unit.h
 )
-target_compile_features(mp-units-core INTERFACE cxx_std_20)
```

Comment:
Don't you think it is a good idea to set the minimum version requirements for the source code to be built in CMake? Yes, I know it is controversial as it overrides the user's setting and possibly results in a higher version than the one set by the user. But for that, we have `check_min_cppstd()` in the conanfile.py to prevent that.

If the user sets C++23 this will not be an issue as the user's setting will overwrite the value here as this one just states the minimum version required to compile those sources.

Reply:
This was moved to `mp-units-core-macros`.